### PR TITLE
Material wall/floor and floor tile name/desc changes

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request

1. Renames material floors to floor instead of plating
2. Removes placeholdery material floor description to make them consistent with regular metal floors which don't have one
3. Changes material wall description to name the material used, to mirror the description of regular metal walls
4. Changes material tiles' name to "floor tile" to be consistent with regular floor tiles
5. Adds custom examine to floor tiles based on their throwforce
5. Buffs material floor tiles' throwforces from 1 to 10 to better showcase the difference between certain materials (e.g. meat, titanium)
5. Radioactive items no longer output a single full stop when examined at a distance
6. Adds warning span to radioactive item message

## Why It's Good For The Game

Polish, consistency, fixes the mistake of calling material floors plating.

## Changelog
:cl: cacogen
fix: Material floors are now correctly called floors and not plating
balance: Buffs material floor tiles' throwforces from 1 to 10 (same as iron) to better showcase the effect of different materials (e.g. meat vs. titanium)
fix: Radioactive items no longer output a single . when examined at a distance
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
@Qustinnus i hope you don't mind

**Comment by Qustinnus** [comment]
No it's a cool PR I just need to look into this examine thing because I'm not sure this is the right way to do it.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 1]
```diff
@@ -335,12 +335,21 @@
...
 	singular_name = "floor tile"
-	desc = "A tile of flooring."
+	throwforce = 5
 	icon_state = "material_tile"
 	turf_type = /turf/open/floor/material
-	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_UPDATE_DESC | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
+
+/obj/item/stack/tile/material/mat_update_desc(mat)
+	switch(throwforce)
```
Comment (Qustinnus) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: I think this could be better handled in Examine()

Comment (Qustinnus) [reply code/game/objects/items/stacks/tiles/tile_types.dm]: also I feel like only adding this info to tiles might seem weird.

File: code/game/turfs/closed/wall/material_walls.dm [hunk 2]
```diff
@@ -17,3 +17,7 @@
 	for(var/i in custom_materials)
 		var/datum/material/M = i
 		new M.sheet_type(src, FLOOR(custom_materials[M] / MINERAL_MATERIAL_AMOUNT, 1))
+
+/turf/closed/wall/material/mat_update_desc(mat)
+	desc = "A huge chunk of [mat] used to separate rooms."
```
Comment (Qustinnus) [inline code/game/turfs/closed/wall/material_walls.dm]: Mmmh alright I see some more use for this for these walls. However. You should remove the material flag and just run the proc, if the object doesn't update the desc based on materials thats up to them, the proc would just return.

File: code/game/objects/items/stacks/tiles/tile_types.dm (hunk 1 above)
Comment (cacogen) [reply code/game/objects/items/stacks/tiles/tile_types.dm]: I just wanted to make it consistent with the long-standing current iron floor tile description without misleading players when these tiles are made of a weak material.

Comment (Qustinnus) [reply code/game/objects/items/stacks/tiles/tile_types.dm]: sure, just make sure this is in examine

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 3]
```diff
@@ -20,6 +20,11 @@
 	pixel_x = rand(-3, 3)
 	pixel_y = rand(-3, 3) //randomize a little
 
+/obj/item/stack/tile/examine(mob/user)
+	..()
+	if(throwforce >= 10)
```
Comment (Qustinnus) [inline code/game/objects/items/stacks/tiles/tile_types.dm]: I feel like there'd be a lack of nuance if at 9 throwforce it already stops being a good throwing weapon, maybe multiple options is better?

Comment (cacogen) [reply code/game/objects/items/stacks/tiles/tile_types.dm]: ok

File: code/datums/components/radioactive.dm [hunk 4]
```diff
@@ -70,9 +70,11 @@
 			out += "[length(out) ? " and it " : "[master] "]seems to be glowing a bit."
 		if(RAD_AMOUNT_HIGH to INFINITY) //At this level the object can contaminate other objects
 			out += "[length(out) ? " and it " : "[master] "]hurts to look at."
-		else
-			out += "."
-	to_chat(user, out.Join())
+	if(LAZYLEN(out))
+		out += "."
+	else
+		return
```
Comment (Rohesie) [inline code/datums/components/radioactive.dm:76]:
```suggestion
	if(!LAZYLEN(out))
		return
	out += "."
```
Bit less convoluted logic with an early return.

File: code/game/objects/items/stacks/tiles/tile_types.dm [hunk 5]
```diff
@@ -14,12 +14,32 @@
...
 	novariants = TRUE
+	var/human_maxHealth = 100
 
 /obj/item/stack/tile/Initialize(mapload, amount)
 	. = ..()
 	pixel_x = rand(-3, 3)
 	pixel_y = rand(-3, 3) //randomize a little
 
+/obj/item/stack/tile/examine(mob/user)
+	..()
```
Comment (Rohesie) [inline code/game/objects/items/stacks/tiles/tile_types.dm:25]: Preserve the return and add the extra text to it, instead of adding a second `to_chat()`

**Pull request merged**